res/database/Default/user/jorgaeff.xml

```diff
@@ -435,9 +503,11 @@
 		<news id="news-jorgaeff-drops-07" type="2" thread_id="news-jorgaeff-drops" creator="8936" created_by="jorgaeff">
 			<title>
 				<de>Benny Jameson zu Gefängnisstrafe verurteilt</de>
+				<en>Benny Jameson sentenced to prison</en>
 			</title>
 			<description>
 				<de>Den geklauten Filmpreis hat er zurück gegeben. Trotzdem wurde Benny Jameson zu einer dreitägigen Gefängnisstrafe verurteilt, die er bereits angetreten hat.</de>
+				<en>The stolen film award he gave back. Nevertheless, Benny Jameson was convicted to a three-day prison sentence, which he has already entered.</en>
```

Comment:
"was returned" or "He gave back"?

Reply:
Yeah, it's sometimes permissible to put the object first for emphasis, but it's far less usual than in German, and maybe not necessary here.
Passive speech wouldn't make it clear who gave it back though.
Changing to normal subject - object sentence.